tokens/src/lib.rs

```diff
@@ -103,6 +104,9 @@ pub trait Trait: frame_system::Trait {
 	/// The currency ID type
 	type CurrencyId: Parameter + Member + Copy + MaybeSerializeDeserialize + Ord;
 
+	/// All non-native currency ids.
+	type AllNonNativeCurrencyIds: Get<Vec<Self::CurrencyId>>;
```

Comment:
we can't have this anymore. at later stage we will have `Erc20(H160)` so it will not be possible to have complete list of CurrencyId